content/container_insights/setup.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Setup Container Insights"
+chapter: false
+weight: 1
+---
+
+There are 2 components that work together to collect Metrics, Logs data for Container Insights in EKS
+
+1. CloudWatch agent
+2. FluentD
+
+#### Application setup
+In this section we will be setting up Container Insights for the ECS Demo app that you created in the **Deploy the Example Microservices** section earlier. If you haven't done that yet, go ahead and deploy the application and come back here again to proceed with Container Insights.
+
+------------------------------------------------------------
+
+#### Setup IAM role
```

Comment:
This entire step can be automated using scripts. You can see here for inspiration https://eksworkshop.com/scaling/deploy_ca/

Reply:
Completed per suggestion